pkgs/tools/security/ghidra/extensions.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, newScope, callPackage, ghidra }:
+
+lib.makeScope newScope (self: {
+  inherit (callPackage ./build-extension.nix { inherit ghidra; }) buildGhidraExtension buildGhidraScripts;
+
```

Comment:
Would you please change this list to be sorted alphabetically

Reply:
Done